src/style.css

```diff
@@ -2,0 +1,24 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background: url('/src/images/star.svg') no-repeat 50%;
```

Comment:
Consider using a more descriptive class name for `.stars__star`. The class name should represent the meaning of the content, not just the style or tag names.